chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -101,12 +101,14 @@
 
 <!ENTITY zotero.preferences.quickCopy.caption				"Quick Copy">
 <!ENTITY zotero.preferences.quickCopy.defaultOutputFormat	"Default Output Format:">
+<!ENTITY zotero.preferences.quickCopy.locale				"Bibliography Locale:">
```

Comment:
Should just be "Locale:" (same as below), since it affects both bibliography and in-text citations